osf/management/commands/import_EGAP.py

```diff
@@ -148,7 +152,7 @@ def main(guid, creator_username):
     egap_assets_path = get_egap_assets(guid, creator_auth)
 
     # __MACOSX is a hidden file created by the os when zipping
-    directory_list = [directory for directory in os.listdir(egap_assets_path) if directory not in ('egap_assets.zip', '__MACOSX')]
+    directory_list = [directory for directory in os.listdir(egap_assets_path) if directory not in ('egap_assets.zip', '__MACOSX') and not directory.startswith('.')]
```

Comment:
Similarly, why not just return the directory that contains the directories you want to use from `get_egap_schemas` rather than doing this again?